tests/test_dialects.py

```diff
@@ -1669,6 +1669,31 @@ def test_snowflake(self):
             "SELECT a FROM test AS t QUALIFY ROW_NUMBER() OVER (PARTITION BY a ORDER BY Z) = 1",
             read="snowflake",
         )
+        self.validate(
+            "SELECT TO_TIMESTAMP(1659981729)",
+            "SELECT TO_TIMESTAMP(1659981729)",
+            read="snowflake",
+        )
+        self.validate(
+            "SELECT TO_TIMESTAMP(1659981729000, 3)",
+            "SELECT TO_TIMESTAMP(1659981729)",
+            read="snowflake",
+        )
+        self.validate(
+            "SELECT TO_TIMESTAMP('1659981729')",
+            "SELECT TO_TIMESTAMP('1659981729')",
+            read="snowflake",
+        )
+        self.validate(
+            "SELECT TO_TIMESTAMP('2013-04-05 01:02:03')",
+            "SELECT TO_TIMESTAMP('2013-04-05 01:02:03', 'yyyy-mm-dd hh24:mi:ss')",
+            read="snowflake",
+        )
+        self.validate(
+            "SELECT TO_TIMESTAMP('04/05/2013 01:02:03', 'mm/dd/yyyy hh24:mi:ss')",
+            "SELECT TO_TIMESTAMP('04/05/2013 01:02:03', 'mm/dd/yyyy hh24:mi:ss')",
+            read="snowflake",
+        )
```

Comment:
add a test for iff and also time conversions?

Reply:
done